CHANGELOG.md

```diff
@@ -92,6 +270,9 @@
   [#12975](https://github.com/rust-lang/cargo/pull/12975)
 - Avoid writing CACHEDIR.TAG if it already exists.
   [#13132](https://github.com/rust-lang/cargo/pull/13132)
+- Accept `?` in the `--package` flag if it's a valid pkgid spec.
```

Comment:
I am not sure if b14c5feeba9ef78bb5d7d995ac64c640abc1c6ad is the correct way to handle backported fixes. Should we keep both PRs in both versions?

Reply:
That is correct, when something is backported it should be moved to the earlier release.